Hunting Queries/W3CIISLog/SuspectedMailBoxExportHostonOWA.yaml

```diff
@@ -1,6 +1,8 @@
 id: a523786c-8382-4029-80e9-f1a7ecd067c1
 name: Suspect Mailbox Export on IIS/OWA
 description: |
+  'Query finds suspicious files accessed on IIS server indicating exfiltration hosting. Observed when exporting mailbox files from OWA servers. Reference: Dark Halo Leverages SolarWinds Compromise.'
```

Comment:
Again, No need of reference string here.

Reply:
Removed reference string and updated description.